module/preference/PreferenceDialog.kt

```diff
@@ -51,7 +51,9 @@ import com.vaticle.typedb.common.collection.Either
 import com.vaticle.typedb.studio.framework.common.theme.Theme
 import com.vaticle.typedb.studio.framework.material.Dialog
 import com.vaticle.typedb.studio.framework.material.Form
+import com.vaticle.typedb.studio.framework.material.Form.ColumnSpacer
 import com.vaticle.typedb.studio.framework.material.Form.Field
+import com.vaticle.typedb.studio.framework.material.Form.RowSpacer
```

Comment:
We import the column and row spacers directly as we no longer have our own PreferenceDialog-specific column and row spacers. This is in line with our importing of other composables from `Form` such as `TextButton` and `Field`.